docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in this issue (https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
```

Comment:
Could change to: [this issue](https://github.com/redux-saga/redux-saga/issues/518)